config/settings/production.py

```diff
@@ -12,10 +12,9 @@
 """
 from __future__ import absolute_import, unicode_literals
 
-from django.utils import six
-
 import logging
 
+from django.utils import six
```

Comment:
Oops, I didn't notice that this bit of "common.py" is not actually common. Arguably it should not go in the common file if it isn't shared? (not saying we should fix it today, but just speaking philosophically)

Reply:
It is different between them (dist vs dist/prod, to get the uglified webpack bundle I'm guessing)